src/services/meetup/index.ts

```diff
@@ -1,17 +1,20 @@
-import { MEETUP_LINK } from "@/const";
+import { MEETUP_RSS_LINK } from "@/const";
 
 export const getMeetupInfo = async () => {
-  const meetupResponse = await fetch(MEETUP_LINK, { cache: "no-store" });
+  const meetupResponse = await fetch(MEETUP_RSS_LINK, { cache: "no-store" });
   const meetupText = await meetupResponse.text();
-  // create an array with all the time elements with the class "text-[#00829B] text-sm font-medium uppercase"
-  const eventList = meetupText.match(
-    /<time class="text-\[#00829B\] text-sm font-medium uppercase">(.*?)<\/time>/g
```

Comment:
I have no idea why this regex wasn't matching. I logged `meetupText` then put it into https://regexr.com with the old regex and it matched perfectly

<img width="1075" alt="image" src="https://github.com/user-attachments/assets/6d352791-df9c-48a1-bae1-209e2d25b4d9">